cli/azd/pkg/azapi/deployments_operations.go

```diff
@@ -13,6 +13,7 @@ import (
 	azdinternal "github.com/azure/azure-dev/cli/azd/internal"
 	"github.com/azure/azure-dev/cli/azd/pkg/account"
 	"github.com/azure/azure-dev/cli/azd/pkg/azsdk"
+	azdCloud "github.com/azure/azure-dev/cli/azd/pkg/cloud"
```

Comment:
Can we import this as "cloud" and not `azdCloud` or do things get broken if we do?

Reply:
There were a couple places where there was ambiguity between `azure-sdk-for-go`'s `cloud` and our `cloud`... Thanks for pointing these out.